lib/galaxy/tools/deps/resolvers/modules.py

```diff
@@ -154,7 +155,10 @@ def __modules(self):
                 yield module_name, module_version
 
     def __module_avail_output(self):
-        avail_command = [self.module_dependency_resolver.modulecmd, 'sh', 'avail']
+        modulecmd = self.module_dependency_resolver.modulecmd
+        if not exists(modulecmd):
```

Comment:
In some module implementations, the `module` command is actually a bash function. E.g. on my cluster:
```
$ which module 
/usr/bin/which: no module in ...
$ declare -f module
module () 
{ 
    BRIGHT='\e[1m';
...
```